exercise/static/exercise/chapter.js

```diff
@@ -213,6 +192,9 @@
 
 			this.loader = this.chapter.cloneLoader();
 
+			if (this.quiz) {
+				this.element.append(this.settings.quiz_progress_selector);
```

Comment:
This doesn't look right. Assuming `this.element` is an element, then append would expect a HTML element and not a string containing `.quiz-submit-progress`.

Reply:
You are right! I removed it